internal/util/kubernetes/dynamic_client.go

```diff
@@ -187,7 +187,7 @@ func CreateCR(
 	if err != nil {
 		return fmt.Errorf("failed to create Resource %s/%s, err=%v", object.Namespace, object.Name, err)
 	}
-	numaLogger.Debugf("successfully created resource %s/%s", object.Namespace, object.Name)
+	numaLogger.Infof("successfully created resource %s/%s", object.Namespace, object.Name)
```

Comment:
want to have at least one "Info" level line for whenever we create the child resource. Previously did this for the `ApplyCRSpec()` function - didn't realize I needed to do it here too